index.html

```diff
@@ -5452,24 +5452,25 @@ <h3>Base Context</h3>
 the referenced files to be modified.
         </p>
         <p>
-Implementations MUST ensure that the base context value, located at
-<code>https://www.w3.org/ns/credentials/v2</code>, matches the following
-SHA-384 digest of the value computed and encoded according to the [[SRI]] definition of `digest`:
-<strong><code>lHKDHh0msc6pRx8PhDOMkNtSI8bOfsp4giNbUrw71nXXLf13nTqNJoRp3Nx+ArVK</code></strong>.
-The base context value matching the digest previously stated can be used to
-implement a local cached copy. It is possible to confirm the
-SHA-384 digest by running the following command from a modern Unix command
-interface line:
-`curl -s https://www.w3.org/ns/credentials/v2 | openssl dgst -sha384 -binary | openssl base64 -A`.
-        </p>
-        <p class="issue" data-number="1177">
-The Working Group is currently discussing what a processor should do if a hash
-value differs from one that is listed in the specification.
-        </p>
-More details regarding this hash encoding method may be found in the <a
+Implementations MUST treat the base context value, located at
+<code>https://www.w3.org/ns/credentials/v2</code>, as already resolved where the
+content of the resource matches the following SHA-384 digest of the value
+computed and encoded according to the [[SRI]] definition of `digest`:
```

Comment:
Hrm... "if" implies that they should try and resolve it rather than never try to resolve it and always use a local copy to resolve it. That's what we're trying to say here -- "Just use a local copy and that local copy MUST match the provided hash". 

I'll think about this some more and try to provide alternative language that's more straight forward.

Reply:
Should `already resolved ` be `already dereferenced` or perhaps `already retrieved`? I ask because elsewhere, "resolution" is being used to describe conversion from "relative URL" to what is often (apparently incorrectly) called "absolute URL".

Reply:
Yeah, perhaps "already retrieved" is the best word we could use for people that don't know the differences between "resolved" and "dereferenced"... to this day, I still have to look up the proper terminology when authoring specs.

Reply:
Hrm, no "if" isn't the right word. The content at that resource will be the hash value, full stop.

Reply:
Fixed in 31bf381baca615410f10cf1830cdad423be96180.